addons/pylons/XEH_preInit.sqf

```diff
@@ -6,4 +6,67 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+[
```

Comment:
We have been putting settings into a file called initSettings.sqf
and then `#include "initSettings.sqf"`